schemas/Intent.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Intent.json",
+    "title": "Intent",
+    "description": "A planned economic flow, which can lead to economic events (sometimes through commitments).",
+    "type": "object",
+    "properties": {
+        "action": {
+            "description": "Relates a process input or output to a verb, such as consume, produce, work, improve, etc.",
```

Comment:
"Relates a process input or output to a verb
s/b
"Relates an intent to a verb ?